modules/ROOT/pages/7.6.0-release-notes.adoc

```diff
@@ -107,85 +182,217 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 [[improvements]]
 == Improvements
 
-{productname} {release-version} also includes the following improvement<s>:
+{productname} {release-version} also includes the following improvements:
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
-
-// CCFR here.
+=== Base64 data was not properly decoded due to unhandled URL-encoded characters.
+// #TINY-9548
 
+=== The `latin` list style type is now recognized as an alias for the `alpha` list style type.
+// #TINY-11515
 
 [[additions]]
 == Additions
 
-{productname} {release-version} also includes the following addition<s>:
-
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
-
-// CCFR here.
+{productname} {release-version} also includes the following additions:
+
+=== Add Labels and Groups for Context Toolbar Buttons
+// #TINY-11095
+
+The release of {productname} {release-version} introduces the ability to organize context toolbar buttons into groups with optional labels or titles. This enhancement, available from {productname} 7.6.0 onward, improves toolbar usability by enabling clearer categorization of buttons.
+
+The `items` object structure now supports defining groups with an optional `name` property for titles or a `label` property for identifying the group. This feature allows developers to create more intuitive and accessible toolbars by visually segmenting functionality.
+
+Example of a context toolbar configuration with groups and labels:
+
+.Example
+[source,js]
+----
+items: [
+  {
+    name: 'Formatting', // Optional, used as the group's title
+    items: [ 'bold', 'italic' ] // Array of registered button names
+  },
+  {
+    label: 'History', // Optional, used as a label for the group
+    items: [ 'undo', 'redo' ] // Array of registered button names
+  },
+]
+----
+
+For more details on configuring context toolbar groups and labels, see: xref:contexttoolbar.adoc#add-labels-and-groups-for-context-toolbar-buttons[Context Toolbar].
+
+=== New `contextsliderform` and `contextsizeinput` context form types.
+// #TINY-11342
```

Comment:
Commenting out this section as this will be updated post release as it requires detailed changes to existing documentation.

```suggestion
// === New `contextsliderform` and `contextsizeinput` context form types.
// #TINY-11342
```

Reply:
@spocke Reminder to update this section also once hotfix DOC-2595 is up.